lib/Bio/KBase/ObjectAPI/BaseObject.pm

```diff
@@ -408,6 +408,8 @@ sub toJSON {
     return $JSON->encode($data);
 }
 
+sub TO_JSON { shift->toJSON( @_ ) }
```

Comment:
The JSON module will automatically use the `TO_JSON` method for serialising an object.